drivers/intc/intc_arm_v8m_nvic/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2024 Bjarki Arge Andreasen
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+zephyr_library_sources(arm_v8m_nvic.c)
+zephyr_linker_sources(ROM_START SORT_KEY 0x0vectors irq-vector-table.ld)
```

Comment:
Will this be an issue for the `CMAKE_LINKER_GENERATOR`?

Reply:
I think so, I will need to look into making it more generic, it is a bit of a hack currently :)